docs/ops/detection/ExperimentalDetectronDetectionOutput_6.md

```diff
@@ -0,0 +1,161 @@
+## ExperimentalDetectronDetectionOutput <a name="ExperimentalDetectronDetectionOutput"></a> {#openvino_docs_ops_detection_ExperimentalDetectronDetectionOutput_6}
+
+**Versioned name**: *ExperimentalDetectronDetectionOutput-6*
+
+**Category**: Object detection
+
+**Short description**: An operation *ExperimentalDetectronDetectionOutput* performs non-maximum suppression to generate 
+the detection output using information on location and score predictions.
+
+**Detailed description**: Operation doing next steps:
+
+1.  Applies deltas to boxes sizes;
```

Comment:
Do you mean `delta_weights`? Could you please elaborate how they are applied?

Reply:
Done.